src/js/utils/ui.js

```diff
@@ -86,15 +86,19 @@ function initInteractionListeners(ui) {
     const listenerOptions = passiveEvents ? { passive } : false;
 
     const interactStartHandler = (e) => {
```

Comment:
This works on the `playerElement` due to the following: https://github.com/jwplayer/jwplayer/blob/master/src/js/view/view.js#L216